java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java

```diff
@@ -468,16 +470,15 @@ static TruthValue evaluatePredicateRange(PredicateLeaf predicate, Object min,
       }
       // in case failed conversion, return the default YES_NO_NULL truth value
     } catch (Exception e) {
-      if (LOG.isWarnEnabled()) {
+      if (LOG.isDebugEnabled()) {
```

Comment:
This isn't ok. We can't take exceptions and make them invisible. We need to avoid the exception.